attachments/models.py

```diff
@@ -122,7 +122,9 @@ def get_field(self, prop):
             Added for use in bootstrap's template tag render_value. Returns tuple of property label and value
         """
         if prop.data_type == 'model' and prop.slug in self.data:
-            return prop.label, prop.model_queryset().get(pk=self.data.get(prop.slug, [])[0])
+            pk = self.data.get(prop.slug, [])[0]
```

Comment:
This will raise an IndexError if the slug isn't in `self.data` since the default is an empty list

Reply:
Brad should clearly be doing these reviews instead of me!

Reply:
Ah ok, I was thinking you wanted to be looped in on all updates here, but will assign to Brad or Noah next time :)